sw/device/lib/ujson/rust/roundtrip_test.rs

```diff
@@ -125,7 +143,12 @@ mod test {
     #[test]
     fn test_fuzzy_maybe() -> Result<()> {
         let before = FuzzyBool(49);
-        let after = roundtrip("fuzzy_bool", &serde_json::to_string(&before)?)?;
+        // The C ujson deserialiser reads the trailing newline
+        // when deserialising this type,
+        // as it has no other way of knowing the last character of the integer.
+        // This means the crc calculation includes the trailing newline,
+        // causing a mismatch with the actual value.
+        let after = roundtrip("fuzzy_bool_no_crc", &serde_json::to_string(&before)?, false)?;
```

Comment:
Well, that's kindof a pain and a basic problem with the way the parser uses getc/ungetc and bare undelimited values like integers.

Would you mind create an issue and leaving a TODO here:
```
// TODO(opentitan#xxxxx): The C ujson deserializer ....
```